service/backend/pchain/indexer/parser.go

```diff
@@ -57,8 +52,6 @@ type parser struct {
 	ctx *snow.Context
 
 	pChainClient client.PChainClient
-
-	genesisTimestamp time.Time
```

Comment:
Once `genesisTimestamp` is not used anymore to check `errParserUninitialized`, we can simply drop it